.github/workflows/ci-pytorch-tests-slow.yml

```diff
@@ -8,7 +8,7 @@ on:
     branches: [master, "release/*"]
     types: [opened, reopened, ready_for_review, synchronize]  # add `ready_for_review` since draft is skipped
     paths:
-      - ".github/workflows/ci-pytorch-test-slow.yml"
+      - ".github/workflows/ci-pytorch-tests-slow.yml"
```

Comment:
Can we have a better name then slow here?

Reply:
It runs tests marked as "slow". What name would you prefer? Its also the usual marker name in other projects

This can be done separately